aspnetcore/performance/caching/middleware.md

```diff
@@ -68,46 +85,50 @@ if (responseCachingFeature != null)
+| Pragma | A `Pragma: no-cache` header in the request produces the same effect as `Cache-Control: no-cache`. This header is overridden by the relevant directives in the `Cache-Control` header, if present. Considered for backward compatibility with HTTP/1.0. |
+| Set-Cookie | The response isn't cached if the header exists. |
+| Vary | The `Vary` header is used to vary the cached response by another header. For example, you can cache responses by encoding by including the `Vary: Accept-Encoding` header, which caches responses for requests with headers `Accept-Encoding: gzip` and `Accept-Encoding: text/plain` separately. A response with a header value of `*` is never stored. |
+| Expires | A response deemed stale by this header isn't stored or retrieved unless overridden by other `Cache-Control` headers. |
+| If-None-Match | The full response is served from cache if the value isn't `*` and the `ETag` of the response doesn't match any of the values provided. Otherwise, a 304 (Not Modified) response is served. |
+| If-Modified-Since | If the `If-None-Match` header isn't present, a full response is served from cache if the cached response date is newer than the value provided. Otherwise, a 304 (Not Modified) response is served. |
+| Date | When serving from cache, the `Date` header is set by the middleware if it wasn't provided on the original response. |
+| Content-Length | When serving from cache, the `Content-Length` header is set by the middleware if it wasn't provided on the original response. |
+| Age | The `Age` header sent in the original response is ignored. The middleware computes a new value when serving a cached response. |
 
 ## Troubleshooting
-If caching behavior is not as you expect, confirm that responses are cacheable and capable of being served from the cache by examining the request's incoming headers and the response's outgoing headers. The conditions by which a response will be cached are listed below.
+If caching behavior isn't as you expect, confirm that responses are cacheable and capable of being served from the cache by examining the request's incoming headers and the response's outgoing headers. The conditions by which a response are cached are listed below.
+
+Enabling logging can help when debugging. The middleware logs caching behavior and when a response is retrieved from cache. See [Logging in ASP.NET Core](xref:fundamentals/logging) for more information on enabling logging.
```

Comment:
See [Logging in ASP.NET Core](xref:fundamentals/logging) for more information ~on enabling logging.~